packages/build/package.json

```diff
@@ -20,7 +20,7 @@
     "debug": "^4.0.1",
     "fs-extra": "^7.0.0",
     "glob": "^7.1.2",
-    "mocha": "^5.1.1",
+    "mocha": "^6.0.0-1",
```

Comment:
As commented in the original PR, I am reluctant to use pre-release versions. Can we wait until `6.0.0` is released please, and then use that version instead?

Reply:
Sure. The PR was created to test out 6.x.